packages/react-server/docs/understanding-rendering.md

```diff
@@ -7,49 +7,66 @@ a "finished" html document.  This guide will help you understand how different
 parts of the rendering lifecycle take place, and how the final document is
```

Comment:
The diff chunking in here isn't great.  Really everything after the first paragraph is new.